populate_sd_card.sh

```diff
@@ -157,9 +188,20 @@ echo "| WARNING! WARNING! WARNING! WARNING! WARNING! WARNING! WARNING! |"
 echo "=================================================================="
 read ans
 
+DEVICE="$1"
+if [ "$DEVICE" = "vision-ai-rzv2l" ] ; then
+	populate_sd_card_rootfs "$2"
+else
+	echo "Unsupported device $DEVICE"
+	exit 1
+fi
+
 DEVICE="$1"
 if [ "$DEVICE" = "smarc-rzg2l" ] ; then
-	populate_sd_card_rootfs_and_fat "$2"
+        populate_sd_card_rootfs_and_fat "$2"
```

Comment:
nitpick: this is a whitespace change; it was a tab here;

Reply:
probably done